Library/ViewModels/RewardPledgeViewModel.swift

```diff
@@ -224,20 +230,29 @@ RewardPledgeViewModelOutputs {
       .map { AppEnvironment.current.currentUser }
       .skipRepeats(==)
 
-    let shippingRules = projectAndReward
-      .switchMap { (project, reward) -> SignalProducer<[ShippingRule], NoError> in
+    let shippingIsLoadingProperty = MutableProperty(false)
```

Comment:
i think we could get rid of the mutable property by leverage the signals we have to determine loading/finished loading, e.g.

```swift
self.shippingIsLoading = Signal.merge(
  projectAndReward.mapConst(true),
  shippingRulesEvent.map { $0.isTerminating }
)
```